src/v2/modules/mod/commands/roles.ts

```diff
@@ -9,16 +9,16 @@ import { ROLES } from '../../roles/consts/roles.js';
 
 const generateButtons = (roles: typeof ROLES | typeof NOTIFY_ROLES) =>
   roles.map(item =>
-    new MessageButton()
+    new ButtonBuilder()
       .setCustomId(`roles🤔toggle🤔${item.name}`)
       .setLabel(item.name)
-      .setStyle('SECONDARY')
+      .setStyle(ButtonStyle.Secondary)
       .setEmoji(item.emoji)
   );
 const chunkAndRowify = pipe<
-  Iterable<MessageButton>,
-  Iterable<MessageActionRow>
->([chunk(5), map(x => new MessageActionRow().addComponents(...x))]);
+  Iterable<ButtonBuilder>,
+  Iterable<ActionRowBuilder<MessageActionRowComponentBuilder>>
+>([chunk(5), map((x: ButtonBuilder[]) => new ActionRowBuilder<MessageActionRowComponentBuilder>().addComponents(...x))]);
```

Comment:
nitpick: can you this be `(actionRow: ButtonBuilder[]) => ...`

or anything that's not `x`